cli/tests/integration_tests.rs

```diff
@@ -1371,6 +1371,11 @@ itest!(_004_set_timeout {
   output: "004_set_timeout.ts.out",
 });
 
+itest!(_004_set_timeout_clear {
+  args: "run --quiet --reload 004_set_timeout_clear.ts",
+  output: "004_set_timeout_clear.ts.out",
+});
+
```

Comment:
I think this test could pass without the change in this PR. We would be happier if we could confirm that it finishes immediately, not take 10 seconds. Is this difficult to check?

Reply:
Good point, I'll rewrite the test

Reply:
Cool :+1: